README.md

```diff
@@ -1,10 +1,22 @@
 # Tic-Tac-Toe
 
-> In this project, we are going to use all the acquired knowledge for Ruby, including OOP (Object Oriented Programming), Modules, Mixins along with all other tools.
+## Instructions
+
+**Goal of the game**
+
+> Tic-tac-toe is a two-player game in which the goal is to create a 3 in-a-row sequence of your symbol inside a 3x3 grid. The sequence can either be vertical, horizontal, or diagonal. Here are some screenshots:
 
 ![screenshot](assets/screenshot1.png)
 
-This first Milestone consisted of setting up the environment and mandatory branches and files to start the development.
+![screenshot](assets/screenshot2.png)
+
+![screenshot](assets/screenshot3.png)
+
+**Rules**
+
+- Each player should take a turn to place the symbol in an empty square.
+- For this implementation, player one will have the symbol "X" and player two the symbol "O".
+- If all squares are occupied by a symbol and there is no 3-in-a-row sequence then the game is considered a tie.
```

Comment:
Instead of saying 3-in-a-row I would say `winning combination` cause you have already described the winning combination in goal of the game section.